curriculum/challenges/_meta/css-box-model/meta.json

```diff
@@ -0,0 +1,198 @@
+{
+  "name": "CSS Box Model",
+  "isUpcomingChange": true,
+  "dashedName": "css-box-model",
+  "order": 11,
```

Comment:
I went through the first couple lessons on this again after fixing those conflicts. After completing part one on this, I get sent to part one on the Picasso painting, then to part 2 here, and then to part 2 on the painting, and so on... Pretty sure it's cause they share this order number. I didn't verify it, but they are both using 11 - seems likely that it's the culprit.

Reply:
Yep. That's the issue - ran in to it with the piano project today.
Will fix in the morning 🙂